k8s/prod/common/rpe/send-letter-service.yaml

```diff
@@ -0,0 +1,32 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: rpe-send-letter-service
+  namespace: rpe
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: rpe-send-letter-service
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: rpe-send-letter-service
+    version: 0.0.9
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/rpe/send-letter-service:latest
+      environment:
+        SMTP_HOST: smtp.office365.com
+        FTP_DOWNTIME_FROM: 16:00
+        FTP_DOWNTIME_TO: 17:00
+        UPLOAD_SUMMARY_REPORT_ENABLED: true
+        ENCRYPTION_ENABLED: true
```

Comment:
is it not enabled by default now ?

Reply:
default is true as well ,just in case  added for production

Reply:
Please remove it if it default already

Reply:
removed